Misc/NEWS.d/next/Library/2018-03-11-08-44-12.bpo-33034.bpb23d.rst

```diff
@@ -0,0 +1,3 @@
+Providing an explicit error message when casting the port property to anything 
+that is not and integer value using ``urlparse()`` and ``urlsplit()``.  Port 
```

Comment:
Just a quick question: "... is not and integer ..." is 'and' a typo here?

Reply:
Yes it is.  Thank you!  Updating.